python/tests/tornado/test_hyperbahn.py

```diff
@@ -54,11 +54,3 @@ def test_request():
             arg3='boo',
             headers={'as': 'qux'},
         )
-
-
-@pytest.mark.gen_test
-def test_register():
```

Comment:
Advertise method is highly tight to autobahn. I tested on dev server with autobahn.
It is not very ideal to unit test here. When we have a mock autobahn, we may test it.